media-libs/x265/x265-1.8.ebuild

```diff
@@ -54,6 +58,8 @@ multilib_src_configure() {
 	if [ "${ABI}" = x86 ] ; then
 		use 10bit && ewarn "Disabling 10bit support on x86 as it does not build (or requires to disable assembly optimizations)"
 		mycmakeargs+=( -DHIGH_BIT_DEPTH=OFF )
+	elif [ "${ABI}" = x32 ] ; then
```

Comment:
If I may add my own nitpick, please use bash `[[ ... ]]`. Then quoting becomes unnecessary:

    elif [[ ${ABI} = x32 ]] ; then

If that wouldn't be a problem, please also change the other conditional.